SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialTestPossession/CrossServerPossessionTest.cpp

```diff
@@ -53,16 +52,19 @@ void ACrossServerPossessionTest::PrepareTest()
 				{
 					AssertTrue(PlayerController->HasAuthority(), TEXT("PlayerController should HasAuthority"), PlayerController);
 					AssertFalse(Pawn->HasAuthority(), TEXT("Pawn shouldn't HasAuthority"), Pawn);
-					PlayerController->UnPossess();
-					PlayerController->RemotePossessOnServer(Pawn);
+					if (!(Pawn->HasAuthority()))
+					{
+						PlayerController->UnPossess();
+						PlayerController->RemotePossessOnServer(Pawn);
+					}
 				}
 			}
 		}
 		FinishStep();
 	});
 
 	AddStep(
-		TEXT("Check test result"), FWorkerDefinition::Server(1),
+		TEXT("Check test result"), FWorkerDefinition::Server(4),
```

Comment:
Could you add an explicit comment that the pawn is expected to be spawned on Server 4 (or rather located in its area)?

Reply:
Yup